platformio.ini

```diff
@@ -72,15 +72,26 @@ build_flags =
     -D LOG_DEBUG_ENABLE=1
     -D CONFIG_LOG_SEVERITY=Logging::LOG_LEVEL_DEBUG
     -lmosquitto
+    -I./lib/Webots/include/c
+    -I./lib/Webots/include/cpp
+    -L./lib/Webots/lib/controller
+    -llibController
 lib_deps =
     HALInterfaces
     ArduinoNative
     HALSim
+    Webots
 lib_ignore =
     HALTarget
     HALTest
 extra_scripts =
+    ./scripts/webots_launcher.py
     pre:./scripts/add_os_specific_build_flags.py

```

Comment:
Script `scripts\add_os_specific_build_flags.py` can now be removed, as it is only necessary for socket connection

Reply:
ok